hazelcast/src/main/java/com/hazelcast/map/QueryResultSizeExceededException.java

```diff
@@ -27,6 +27,10 @@
  */
 public class QueryResultSizeExceededException extends HazelcastException {
 
+    public QueryResultSizeExceededException(String message) {
```

Comment:
This missing constructor was the main cause for the timeout issue of the nightly build.